### PR TITLE
[trust] Addition of trust Sylvie UC

### PR DESCRIPTION
**Pull request by Omnione**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Adds the trust Sylvie UC to a somewhat decent state

The Indi spell selection gambit for macc is not perfect but its enough to get let her decide if the player needs more macc or not.
All the main stuff is there, No Nott ws yet same as Apururu, so thats a TODO

## Steps to test these changes

Tested on multiple levels and jobs

### CODE REVIEW

**Comment by zach2good** [comment]
Coooool!

File: scripts/globals/gambits.lua [hunk 1]
```diff
@@ -75,6 +75,8 @@ ai.select =
     BEST_AGAINST_TARGET = 6,
     BEST_SAMBA          = 7,
     HIGHEST_WALTZ       = 8,
+    ENTRUSTED           = 9,
+    BESTINDI            = 10,
```
Comment (zach2good) [inline scripts/globals/gambits.lua]: If this is two words, can it be `BEST_INDI`?

File: src/map/ai/helpers/gambits_container.cpp [hunk 2]
```diff
@@ -341,6 +341,25 @@ namespace gambits
                         //    controller->Cast(target->targid, static_cast<SpellID>(spell_id.value()));
                         //}
                     }
+                    else if (action.select == G_SELECT::BESTINDI)
```
Comment (zach2good) [inline src/map/ai/helpers/gambits_container.cpp]: Same here: `BEST_INDI`

**Review by zach2good — APPROVED** [review]
Looks good 👍 

We don't have unity rankings and stuff do we? I haven't been paying attention. How do you use her?

File: scripts/globals/gambits.lua (hunk 1 above)
Comment (Omnione) [reply scripts/globals/gambits.lua]: yes you'r right, it should be

**Pull request merged**